frontend/src/views/OfferView.vue

```diff
@@ -79,27 +80,27 @@ async function onSaveOffer() {
 
   router.push(`${ROUTES.ORDER.path}/${route.params.order_id}/edit`);
 }
-function onItemDelete(item: OfferItemCreate) {
-  offerItemsArray.value = offerItemsArray.value.filter((element) => element.id !== item.id);
+function onItemDelete(id: number) {
+  offerItemsArray.value = offerItemsArray.value.filter((element) => element.id !== id);
 }
 const allItemsSum = computed(() => {
-  let resultNetto = 0;
-  let resultBrutto = 0;
-  let resultUst = 0;
+  let amountNet = 0;
+  let amountGross = 0;
+  let amountVat = 0;
 
   for (let i = 0; i < offerItemsArray.value.length; i++) {
     const amount = offerItemsArray.value[i].amount ?? 0;
     const price = offerItemsArray.value[i].price ?? 0;
 
-    resultNetto += amount * price;
-    resultBrutto += amount * price * (1 + getVatRate({ isoDate: offerInfo.value.offered_at }));
-    resultUst = resultBrutto - resultNetto;
+    amountNet += round(amount * price, 2);
+    amountGross += amountNet * (1 + getVatRate({ isoDate: offerInfo.value.offered_at }));
+    amountVat = round(amountGross - amountNet, 2);
```

Comment:
Maybe you can even only calculate the amountNet if you prefer and use that below.

```suggestion
    amountNet += round(amount * price, 2);
    amountGross += round(amountNet * (1 + getVatRate({ isoDate: offerInfo.value.offered_at })),2);
    amountVat = round(amountGross - amountNet, 2);
```